docs/source/getting-started/tabular-data/agg-and-filter.ipynb

```diff
@@ -0,0 +1,481 @@
+    "        (\"YEAR\", \"QUARTER\",): dp.polars.Margin(public_info= \"keys\", max_partition_length=estimated_max_partition_len, max_partition_contributions=1),\n",
+    "        (): dp.polars.Margin(public_info= \"lengths\",max_partition_length=estimated_max_partition_len, max_num_partitions=1),\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Singular Variable *`group_by`* "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To demonstrate the `group_by` method, compute the total number of respondents per year. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div><style>\n",
+       ".dataframe > thead > tr,\n",
+       ".dataframe > tbody > tr {\n",
+       "  text-align: right;\n",
+       "  white-space: pre-wrap;\n",
+       "}\n",
+       "</style>\n",
+       "<small>shape: (10, 2)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>YEAR</th><th>Actual Count</th></tr><tr><td>i64</td><td>u32</td></tr></thead><tbody><tr><td>2004</td><td>16491</td></tr><tr><td>2005</td><td>16460</td></tr><tr><td>2006</td><td>16291</td></tr><tr><td>2007</td><td>16838</td></tr><tr><td>2008</td><td>16774</td></tr><tr><td>2009</td><td>19998</td></tr><tr><td>2010</td><td>24081</td></tr><tr><td>2011</td><td>24776</td></tr><tr><td>2012</td><td>24952</td></tr><tr><td>2013</td><td>23339</td></tr></tbody></table></div>"
+      ],
+      "text/plain": [
+       "shape: (10, 2)\n",
+       "┌──────┬──────────────┐\n",
+       "│ YEAR ┆ Actual Count │\n",
+       "│ ---  ┆ ---          │\n",
+       "│ i64  ┆ u32          │\n",
+       "╞══════╪══════════════╡\n",
+       "│ 2004 ┆ 16491        │\n",
+       "│ 2005 ┆ 16460        │\n",
+       "│ 2006 ┆ 16291        │\n",
+       "│ 2007 ┆ 16838        │\n",
+       "│ 2008 ┆ 16774        │\n",
+       "│ 2009 ┆ 19998        │\n",
+       "│ 2010 ┆ 24081        │\n",
+       "│ 2011 ┆ 24776        │\n",
+       "│ 2012 ┆ 24952        │\n",
+       "│ 2013 ┆ 23339        │\n",
+       "└──────┴──────────────┘"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "count_year_actual = (\n",
+    "    df.group_by(\"YEAR\").agg(\n",
+    "        pl.len().alias(\"Actual Count\")\n",
+    "    ).sort(\"YEAR\")\n",
+    ").collect()\n",
+    "count_year_actual"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now to get the differentially private statistics, add `dp.noise` after the aggregate function is specified and `.release` after the entire query before `.collect`. \n",
+    "\n",
+    "Calling `.release` is always the final step in compiling your differentially private data in a usable form and ensuring it is compliant with differential privacy guarantees. \n",
```

Comment:
I don't think "compiling" is the right word in this context. Also "in a usable form" seems like elusive language. Why is it usable? Usability seems like a different concept from "safety" or "privacy" which is the relevant concept being discussed right here.

Reply:
what do you recommend adding?